server/operations/src/main/java/org/kaaproject/kaa/server/operations/service/akka/actors/io/EncDecActorMessageProcessor.java

```diff
@@ -15,12 +15,14 @@
  */
 package org.kaaproject.kaa.server.operations.service.akka.actors.io;
 
+import java.io.IOException;
 import java.security.GeneralSecurityException;
 import java.security.PublicKey;
 import java.text.MessageFormat;
 import java.util.Map;
 import java.util.Set;
 
+import org.kaaproject.kaa.common.channels.protocols.kaatcp.messages.ConnAck;
```

Comment:
This is not correct. EncDecActor should not know about concrete implementations of transport.